nifi-registry-core/nifi-registry-web-api/src/main/java/org/apache/nifi/registry/web/security/NiFiRegistrySecurityConfig.java

```diff
@@ -88,7 +88,8 @@ public NiFiRegistrySecurityConfig() {
     @Override
     public void configure(WebSecurity webSecurity) throws Exception {
         // allow any client to access the endpoint for logging in to generate an access token
-        webSecurity.ignoring().antMatchers( "/access/token/**");
+        webSecurity.ignoring().antMatchers("/access/config", "/access/token", "/access/kerberos",
+                "/access/oidc/exchange", "/access/oidc/callback", "/access/oidc/request", "/access/token/identity-provider");
```

Comment:
I'm not sure we have "/access/config", I only see "/config" which maps the ConfigResource and returns information about which type of user-group-provider and authorizer are available, which could probably go either way on whether that is exposed to unauthenticated users. I don't think it is needed by anything until after you login, but its also not exposing anything important.

Reply:
Yeah good point, I'll verify all of these patterns as some appear different to NiFi. We want this list to be concise.